pyproject.toml

```diff
@@ -72,6 +72,12 @@ shapely = { version = ">=1.7.0", optional = true } # groups = ["user", "all"]
 
 pylibjpeg-libjpeg = { version = ">=1.3.2", optional = true } # groups = ["user", "all"]
 
+# AI
+anthropic = { version = ">=0.25.3", optional = true } # groups = ["user", "all"]
```

Comment:
~0.25.3, depending on how stable you think the package API is.

Reply:
I have actually swapped out of using the beta part of the API and instead opted to use the stable API. So I'm actually comfortable leaving this one only lower pinned.